diff --git a/modules/Package.swift b/modules/Package.swift
index a717745a..026b638f 100644
--- a/modules/Package.swift
+++ b/modules/Package.swift
@@ -7,7 +7,7 @@ let package = Package(
     name: "modules",
     defaultLocalization: "en",
     platforms: [
-        .iOS(.v16)
+        .iOS("16.4")
     ],
     products: [
         .library(name: "About", targets: ["About"]),
@@ -105,12 +105,12 @@ let package = Package(
         .package(url: "https://github.com/pointfreeco/swift-case-paths", from: "1.7.2"),
         .package(url: "https://github.com/pointfreeco/swift-url-routing", from: "0.6.2"),
         .package(url: "https://github.com/zcash-hackworks/MnemonicSwift", from: "2.2.5"),
-        .package(url: "https://github.com/valargroup/zcash-swift-wallet-sdk.git", branch: "shielded-vote"),
+        .package(name: "zcash-swift-wallet-sdk", path: "../../zcash-swift-wallet-sdk"),
         .package(url: "https://github.com/flexa/flexa-ios.git", exact: "1.1.4"),
         .package(url: "https://github.com/pacu/zcash-swift-payment-uri", from: "1.0.1"),
         .package(url: "https://github.com/airbnb/lottie-spm.git", from: "4.5.2"),
         .package(url: "https://github.com/KeystoneHQ/keystone-sdk-ios/", from: "0.0.1"),
-        .package(url: "https://github.com/mgriebling/BigDecimal.git", from: Version(stringLiteral: "2.2.3")),
+        .package(path: "../../BigDecimal"),
         .package(url: "https://github.com/siteline/swiftui-introspect", from: "26.0.0"),
         .package(url: "https://github.com/pointfreeco/swift-navigation", from: "2.5.1"),
         .package(url: "https://github.com/liamnichols/xcstrings-tool-plugin", from: "1.2.0")
