diff --git a/Cargo.toml b/Cargo.toml
index 7fdb691b..7bba68b4 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -5,3 +5,9 @@ members = [
     "vote-commitment-tree-client",
 ]
 resolver = "2"
+
+[patch.crates-io]
+orchard = { path = "../orchard" }
+voting-circuits = { path = "../voting-circuits/voting-circuits" }
+imt-tree = { path = "../vote-nullifier-pir/imt-tree" }
+pir-client = { path = "../vote-nullifier-pir/pir/client" }
