diff --git a/Cargo.toml b/Cargo.toml
index 954b4c9e..16a0b4b2 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -85,6 +85,13 @@ voting-circuits = { version = "0.3.1", features = ["share-reveal"] }
 zcash_keys = { version = "0.13", features = ["orchard"] }
 incrementalmerkletree = { version = "0.8", default-features = false }
 
+[patch.crates-io]
+orchard = { path = "../orchard" }
+voting-circuits = { path = "../voting-circuits/voting-circuits" }
+zcash_voting = { path = "../zcash_voting/zcash_voting" }
+imt-tree = { path = "../vote-nullifier-pir/imt-tree" }
+pir-client = { path = "../vote-nullifier-pir/pir/client" }
+
 [build-dependencies]
 bindgen = "0.72"
 cbindgen = "0.29"
